src/torch/gradchecks.py

```diff
@@ -0,0 +1,156 @@
+import os
+import numpy
+# Import the PET reconstruction engine
+import sirf.STIR as pet
+# Set the verbosity
+pet.set_verbosity(1)
+# Store temporary sinograms in RAM
+pet.AcquisitionData.set_storage_scheme("memory")
+import sirf
+msg = sirf.STIR.MessageRedirector(info=None, warn=None, errr=None)
+from sirf.Utilities import examples_data_path
+import matplotlib.pyplot as plt
+import torch
+# set deterministic computing
+torch.use_deterministic_algorithms(True)
+
+from SIRF_torch import *
+# -	Gradient checks
+# -	Adjointness checks
+
+def get_pet_2d():
+    pet_data_path = examples_data_path('PET')
+    pet_2d_raw_data_file = existing_filepath(pet_data_path, 'thorax_single_slice/template_sinogram.hs')
+    pet_2d_acq_data = AcquisitionData(pet_2d_raw_data_file)
```

Comment:
pretty weird you don't need `pet` here. Presumably because of `from sirf.STIR import *`. I don't like it though.